port/ztpf/omrosdump_helpers.c

```diff
@@ -424,10 +424,7 @@ ztpfBuildCoreFile(void *argv_block)
 	sprintf((char *)workSpace(arg), "core.%lX.%d", dibPtr->dstckf,
 			dumpSiginfo(arg)->si_pid);
 	{
-		int pathLen = strlen(pathName);
-		char ebcdicPath[pathLen + 1];
-		a2e_len(pathName, ebcdicPath, strlen(pathName));
-		int fsystype = pathconf(ebcdicPath, _TPF_PC_FS_TYPE);
+		int fsystype = pathconf(pathName, _TPF_PC_FS_TYPE);
```

Comment:
Should all a2e conversions be removed from ztpf? Just asking purely out of curiosity.

Reply:
z/TPF has the same issue translation issue as z/OS (both being ebcdic platforms).  To assist with porting middleware I think we do the same as z/OS where we redefine standard functions to an atoe_* version (eg. atoe_printf) and we link in a library which implements those functions which simply translates string/char parameters or returns as required.  Our a2e environment is still progressing, there still are a handful of functions we need to implement.

In the future we are looking to do this in a better way that works better with C++, but for now we maintaining the same strategy.